indra/CMakeLists.txt

```
@@ -29,7 +29,7 @@ else()
   set( USE_AUTOBUILD_3P ON )
 endif()
 
-set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD 23)
```

I think maybe we could take this PR if we made this optional and allow it to be set from the cmake/autobuild command line.  For example:

```suggestion
if (NOT DEFINED CMAKE_CXX_STANDARD)
  set(CMAKE_CXX_STANDARD 20)
endif()
```

---

https://github.com/secondlife/viewer/pull/3363/commits/0c4b3d4d7c4cb3b979a9e4f0a581fc19deeda4fd